lib/setup.js

```diff
@@ -4,7 +4,7 @@ var PouchDB = require("./constructor");
 var utils = require('./utils');
 var EventEmitter = require('events').EventEmitter;
 PouchDB.adapters = {};
-PouchDB.preferredAdapters = require('./adapters/preferredAdapters.js');
```

Comment:
Another change was to how `preferredAdapters` work. Instead of explicitly setting this as a list of `['idb', 'websql']` in the browser and `['leveldb']` in Node.js, we can just add them to the array when we call `PouchDB.adapter()`.

The only change I had to make to `PouchDB.adapter()` was a third argument, `addToPreferredAdapters`, so we could avoid doing this for the http/https adapters. But this is an API change I wanted anyway, because I would like it for third-party adapter plugins such as `socket-pouch`. Eventually we should probably document the `PouchDB.adapter()` API, but for now it's still "internal."